pages/api/apiCalls.js

```diff
@@ -0,0 +1,7 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+
+export const getGames = async () => {
+  const promise =  await fetch(`https://api.boardgameatlas.com/api/search?order_by=popularity&client_id=REACT_APP_KEYCODE`)
```

Comment:
Opportunity to clean up `index.js` a little by bringing in just the array instead of array as property of an object (See #12 )